capi/VideoDecoderCapi.h

```diff
@@ -20,8 +20,8 @@
 #ifndef __ENABLE_CAPI__
 #define __ENABLE_CAPI__ 1
 #endif
-#include "interface/VideoDecoderDefs.h"
-#include "interface/VideoCommonDefs.h"
+#include "VideoDecoderDefs.h"
+#include "VideoCommonDefs.h"
```

Comment:
Right, this is the fix that we need.  This and the other .h file.
capi/VideoDecoderCapi.h 
capi/VideoEncoderCapi.h 
The other changes are just to get everything to build after this header change.